app/helpers/genres_helper.rb

```diff
@@ -0,0 +1,8 @@
+module GenresHelper
+  def get_genres_for_filter
+    result = [[t("select_genre"), nil]]
+    genres = Genre.order_by_name
+    genres.map{|genre| result.push([genre.name, genre.id])}
```

Comment:
bỏ hết
chỉ cần 1 dòng 
```suggestion
    Genre.order_by_name.pluck(:name, :id).unshift([t("select_genre"), nil])
```